common/db/job_store/dynamo.go

```diff
@@ -73,9 +69,34 @@ func (d *DynamoJobStore) UpdateJobStatus(jobID string, status types.JobStatus) e
 }
 
 func (d *DynamoJobStore) SetJobMeta(jobID string, meta map[string]string) error {
+	// ensure we perform a consistent read after a write
+	defer d.setConsistentRead(true)
+
 	if err := d.table.Update("JobID", jobID).Set("Meta", meta).Run(); err != nil {
 		return err
 	}
 
 	return nil
 }
+
+func (d *DynamoJobStore) SelectAll() ([]*models.Job, error) {
+	defer d.setConsistentRead(false)
```

Comment:
In general I'd do everything possible to avoid fetching all items from a DynamoDB table unless I knew that DynamoDB table had fewer rows than the provisioned capacity.